transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/service/PdfReportGeneratorService.java

```diff
@@ -0,0 +1,11 @@
+package com.qpay.transactionhistorymanager.service;
+
+import com.qpay.libs.models.PdfReportInfo;
+import java.io.IOException;
+
+public interface PdfReportGeneratorService {
```

Comment:
```suggestion
public interface ReportGeneratorService {
```
Today `pdf`, tomorrow `doc`. Don't name like this high level classes. You can name class with `pdf` if it's really work with pdf, for example your implementation